third_party/nvfuser/csrc/ir_builder.cpp

```diff
@@ -11,6 +11,9 @@
 namespace nvfuser {
 
 Val* IrBuilder::newScalar(DataType dtype) {
+  if (std::holds_alternative<PointerOf>(dtype.type)) {
```

Comment:
I wonder why this is not:

`if (isPointerType(dtype.type)) {`

Reply:
fixed